src/main/java/teammates/storage/entity/StudentProfile.java

```diff
@@ -33,8 +37,8 @@
 
     private String nationality;
 
-    /* only accepts "male", "female" or "other" */
-    private String gender;
+    @IgnoreLoad
```

Comment:
I'm not very familiar with Objectify. What does this annotation do here?

Reply:
Sir This [https://realjenius.com/2013/10/25/objectify-entity-subclass-migration/](url)  this will help you a lot in understanding Objectify